libs/template/spec/render-strategies/render-strategies/local.strategy.spec.ts

```diff
@@ -0,0 +1,88 @@
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { mockConsole } from '@test-helpers';
+import { getStrategies } from '@rx-angular/template';
+import { getMockStrategyConfig } from '../../fixtures';
+import { from, of } from 'rxjs';
+import { fakeAsync, tick } from '@angular/core/testing';
+import createSpy = jasmine.createSpy;
+
+const strategyName = 'local';
+
+describe('local Strategy', () => {
+  // beforeAll(() => mockConsole());
+
+  it('should be present in strategies map', () => {
+    const strategy = getStrategies(getMockStrategyConfig())[strategyName];
```

Comment:
You just generalized `getMockStrategyConfig` :).  Can this `getStrategies(getMockStrategyConfig())` be a single fixture/helper?

Reply:
Unfortunately, I have to create in the test as I need a new instance to count the spy calls.